release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -438,6 +438,10 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-networking_{context}"]
 === Networking
 
+[id="ocp-4-17-user-defined-network"]
+==== New UserDefinedNetwork CRD for creating primary and secondary networks (Technology Preview)
+With {product-title} {version}, users can create multiple networks and declare them as primary or secondary through the new `UserDefinedNetwork` (UDN) custom resource definition (CRD). With UDN, Users can secure traffic between namespaces without configuring and managing complex network policies. For more information, see xref:./networking/multiple_networks/will add later when docs are live.
```

Comment:
I know the heading says it, but maybe would be worth also mentioning in the paragraph that this is a TP feature?